### PR TITLE
feat: spinner for execute SQL

### PR DESCRIPTION
**Pull request by AdiGajbhiye**
### Problem

- Showing hint message above cancel button while loading in query result panel 
- Hint message is selected randomly from list and is shown once every hour.

### Solution

Describe the implemented solution. Add external references if needed.

### Screenshot/Demo

A picture is worth a thousand words. Please highlight the changes if applicable.

### How to test

- Steps to be followed to verify the solution or code changes
- Mention if there is any settings configuration added/changed/deleted

## Checklist

- [ ] I have run this code and it appears to resolve the stated issue
- [ ] `README.md` updated and added information about my change

### CODE REVIEW

File: query_panel/main.js [hunk 1]
```diff
@@ -20,6 +20,38 @@ async function cancelQuery() {
 
 const DEFAULT_HEIGHT = 455;
 
+const HINT_VISIBILITY_DELAY = 1 * 60 * 60 * 1000; // 1 hr
```
Comment (anandgupta42) [inline query_panel/main.js:23]: @AdiGajbhiye Can we make this a hidden config?

File: query_panel/main.js [hunk 2]
```diff
@@ -419,6 +472,21 @@ const app = createApp({
           this.focusPreviewPane();
           this.loading = true;
           this.timeExecution();
+          const now = Date.now();
+          this.showHint = false;
+          if (this.lastHintTimestamp + HINT_VISIBILITY_DELAY < now) {
+            this.showHint = true;
+            this.lastHintTimestamp = now;
+            HINTS.sort(() => Math.random() - 0.5);
+            executeCommand("setContext", {
```
Comment (mdesmet) [inline query_panel/main.js:481]: Should we not await setContext?

```suggestion
            await executeCommand("setContext", {
```

**Review by mdesmet — COMMENTED** [review]
LGTM

**Review by mdesmet — COMMENTED** [review]
LGTM

File: query_panel/main.js (hunk 2 above)
Comment (AdiGajbhiye) [reply query_panel/main.js:481]: no. later actions doesn't depend on its result. And assuming it will successfully complete.

**Pull request merged**